mailet/standard/src/test/java/org/apache/james/transport/mailets/RemoveMimeHeaderByPrefixTest.java

```diff
@@ -124,6 +131,9 @@ public void exactMatchOfPrefixShouldBeAllowed() throws MessagingException {
 
         mailet.service(mail);
 
+        assertThat(new MimeMessageUtils(mail.getMessage()).toHeaderList())
```

Comment:
I have only one assertThat, but need several method call to fully describe the state. I don't see why it would be wrong or hurt readability.